Kempe/Kempe.txt

```diff
@@ -0,0 +1,145 @@
+
+This code is contributed by Riyazul555
+
+
+Time Complexity = O(N^2)
```

Comment:
We store the graph using adjacency lists, so that would give us a space complexity of O(N+M), where N is the number of vertices and M is the number of edges.

Kempe's Algorithm uses an [heuristic approach](https://en.wikipedia.org/wiki/Kempe_chain) for determining the [chromatic number](https://www.geeksforgeeks.org/chromatic-number-of-a-graph-graph-colouring/) of an undirected graph, therefore the worst case time complexity would be O(2^N). WDYT?

Please update the complexities and also mention what each symbol used in the complexities denote.